src/renderers/webgl/WebGLRenderLists.js

```diff
@@ -130,10 +130,10 @@ function WebGLRenderList() {
 
 	}
 
-	function sort() {
+	function sort( costumeOpaqueSort, costumeTransparentSort ) {
```

Comment:
I think you mean "custom", not "costume", here?

Reply:
Obviously, apologize.